torch/_export/non_strict_utils.py

```diff
@@ -199,26 +199,71 @@ def fakify(
             "To register a constant input, use torch.utils._pytree.register_constant"
         )
 
+    # Handle traceable wrapper subclasses
+    # TODO tmanlaibaatar (this still doesn't faithfully reflect dynamo one)
+    from torch._dynamo.source import AttrSource
+    from torch.fx.experimental.symbolic_shapes import SubclassSymbolicContext
+    from torch.utils._python_dispatch import is_traceable_wrapper_subclass
+
+    if is_traceable_wrapper_subclass(t):
+        # Get symbolic context for outer tensor
+        outer_context = _create_symbolic_context_for_tensor(
```

Comment:
what does specifying dynamic shapes for tensor subclasses look like? could you add a test or add some casing here to error if we try to have dynamic shapes with tensor subclasses?